compiler/testData/diagnostics/tests/thisAndSuper/unqualifiedSuper/unqualifiedSuperWithAbstractMembers.fir.kt

```diff
@@ -25,7 +25,7 @@ class B : A(), I {
     override val y: Int = super.y
 
     override fun foo(): Int {
-        super.foo()
+        super.<!ABSTRACT_SUPER_CALL!>foo()<!>
         return super.x
```

Comment:
Note that old front-end reports the same diagnostic here, and you are not. You should check also qualified access expressions to achieve this.

Reply:
You're right. I should've paid attention to it.